babeld/babeld.c

```diff
@@ -304,6 +304,12 @@ void babel_clean_routing_process(void)
     flush_all_routes();
     babel_interface_close_all();
 
+    /* Clean babel config */
+    diversity_kind = DIVERSITY_NONE;
+    diversity_factor = BABEL_DEFAULT_DIVERSITY_FACTOR;
+    resend_delay = BABEL_DEFAULT_RESEND_DELAY;
+    smoothing_half_life = BABEL_DEFAULT_SMOOTHING_HALF_LIFE;
```

Comment:
By the way, shouldn't we use `change_smoothing_half_life()`? I see it's also used to calculate `two_to_the_one_over_hl`.

Reply:
You are right, that's the way to do it. Just fixed that.